org.mixedrealitytoolkit.uxcore/Button/PressableButton.cs

```diff
@@ -279,6 +317,28 @@ protected override void Reset()
             ApplyRequiredSettings();
         }
 
+        /// <summary>
+        /// A Unity event function that is called when the script is loaded.
+        /// </summary>
+        protected virtual void Start()
+        {
+            canvasElementRoundedRect = gameObject.FindAncestorComponent<CanvasElementRoundedRect>();
```

Comment:
Why can't we just disable the enable FrontPlate game objects?  Why do we need to target the components?

Reply:
If we need to target the components, then we should either pass those in as serialized fields.

Reply:
The GameObject has other components that contributed to the button's visual appeal that I didn't want to lose hence I only targeted the CanvasElementRoundedRect, also the experiments to find rendering bottlenecks showed the CanvasElementRoundedRect was a main bottleneck.  Resolving this conversation as the new path is to create an array of component where the user can decide which will be dynamic based on proximity, as per updated User Story.